app/views/hostgroups/index.html.erb

```diff
@@ -16,6 +17,9 @@
           <%= label_with_link(hostgroup, 150, authorizer) %>
         </td>
         <td>
+          <%= link_to hostgroup.hosts_count, hosts_path(:search => "hostgroup = #{hostgroup}") %>
```

Comment:
Prefer searching for hosts by hostgroup_fullname or title, in case the same host group name is used in different parts of the ancestry.